interview-questions.md

```diff
@@ -6,25 +6,34 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: OT
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: a parameter is a variable in our function. 
+
+example: const helloMyNameIs = (student) student would be our parameter
+
+an argument is a value given.
+
+example: any name that we try to put in our parameter would be an argument
+helloMyNameIs(Oscar)
```

Comment:
Great use of an example, I would provide examples like this in all of your answers